template-parts/content/content-none.php

```diff
@@ -27,7 +27,7 @@
 
 		<?php else : ?>
 
-			<h1 class="page-title"><?php esc_html_e( 'Nothing Found', 'twentytwentyone' ); ?></h1>
+			<h1 class="page-title"><?php esc_html_e( 'Nothing here', 'twentytwentyone' ); ?></h1>
```

Comment:
Should this be `Nothing Here`? (question, not suggestion)

Reply:
I don't think so https://make.wordpress.org/docs/handbook/documentation-team-handbook/handbooks-grammar-guide/#capitalization